wordpress-mobile/WordPress-Android
# Add prompt to update site title

Fixes #13786

This PR adds the `activeTask` to the quick start repository. This requires the repository to now expose a model containing both the categories and the active task. The active task is passed to the site info block builder which uses it to show the quick start focus point. I've reused the previous component but for the focus point but it's now added directly to the layout which feels much cleaner than adding it dynamically. I couldn't figure out how to start the focus point animation immediately after setting it's visibility/adding it to the layout so instead there is a method which sets visibility and starts the animation. 

I'll add the rest of the functionality like the snackbar and the functionality to hide the focus point in the following PR.

To test:
- Turn the my site flag on and restart the app
- Click on the "Update site title" task in the first quick start card.
- Notice the quick start focus point appears over the Site title


![update_quick_start_title_2](https://user-images.githubusercontent.com/1079756/104923932-0d21f980-599d-11eb-988f-85b2af875c49.gif)



PR submission checklist:
